WORKSPACE

```diff
@@ -413,13 +417,18 @@ hazel_repositories(
             hazel_hackage("shake", "0.17.8", "ade4162f7540f044f0446981120800076712d1f98d30c5b5344c0f7828ec49a2") +
             hazel_hackage("filepattern", "0.1.1", "f7fc5bdcfef0d43a793a3c64e7c0fd3b1d35eea97a37f0e69d6612ab255c9b4b") +
             hazel_hackage("terminal-progress-bar", "0.4.0.1", "c5a9720fcbcd9d83f9551e431ee3975c61d7da6432aa687aef0c0e04e59ae277") +
+            hazel_hackage("rope-utf16-splay" , "0.2.0.0", "83d1961bf55355da49a6b55d6f58d02483eff1f8e6df53f4dccdab1ac49e101d") +
             hazel_hackage(
                 "unix-compat",
                 "0.5.1",
                 "a39d0c79dd906763770b80ba5b6c5cb710e954f894350e9917de0d73f3a19c52",
                 patches = ["@com_github_digital_asset_daml//bazel_tools:unix-compat.patch"],
-            ),
-        pkgs = packages,
+            ) +
+            # This is a special version of Haskell LSP without GPL dependencies
+            [ ("haskell-lsp", {"url": "https://github.com/DavidM-D/haskell-lsp/archive/{}.zip".format(HASKELL_LSP_COMMIT), "sha256": "7d706fbc1beb49a9345083d3eadb5e311936a2a8449e7765a65aa7d298dff9d6", "stripPrefix": "haskell-lsp-{}".format(HASKELL_LSP_COMMIT)})
```

Comment:
What needs to happen for us to switch to upstream apart from https://github.com/alanz/haskell-lsp/pull/70 being merged?

Reply:
This PR needs to be resolved https://github.com/alanz/haskell-lsp/pull/70, there's some LSP discussion going on, perhaps it could be resolved by the release of ropey, but our patches are pretty tiny at the moment. I think I'm also going to try and upstream our instances to make the diff even smaller

Reply:
Can we fork this to the digital-asset namespace please, and use it from there.